docs/document_loader.ipynb

```diff
@@ -4,72 +4,120 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Google DATABASE\n",
-    "\n",
-    "[Google DATABASE](https://cloud.google.com/DATABASE).\n",
-    "\n",
-    "Load documents from `DATABASE`."
+    "# Bigtable\n",
+    "[Bigtable](https://cloud.google.com/bigtable) is a key-value and wide-column store, ideal for fast access to structured, semi-structured, or unstructured data.\n"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Pre-reqs"
+    "## Setting up\n",
+    "\n",
+    "To run this notebook, you will need a Google Cloud Project, a Bigtable instance, and Google credentials."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "tags": []
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "%pip install PACKAGE_NAME"
+    "%pip install langchain-google-bigtable"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Querying for Documents from Bigtable\n",
+    "For more details on connecting to a Bigtable table, please check the [Python SDK documentation](https://cloud.google.com/python/docs/reference/bigtable/latest/client)."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
-   "metadata": {
-    "tags": []
-   },
+   "execution_count": null,
+   "metadata": {},
    "outputs": [],
    "source": [
-    "from PACKAGE import LOADER"
+    "from langchain_google_bigtable import BigtableLoader\n",
+    "\n",
+    "instance_id = \"my_instance\"\n",
+    "table_id = \"my_table\""
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Basic Usage"
+    "## Create the Loader"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "loader = BigtableLoader(\n",
+    "    instance_id,\n",
+    "    table_id,\n",
+    ")"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Load from table"
+    "You can fetch the documents by calling the `load` method of the loader. It will return a list with all the documents. If you want to avoid this blocking call, you can call `lazy_load` method that returns an Iterator."
```

Comment:
I think we should encourage lazy_load -- load is only around for legacy reasons IIUC

Reply:
Done